frontend/test/metabase/scenarios/dashboard/chained-filters.cy.spec.js

```diff
@@ -109,6 +109,7 @@ function createDashboard({ dashboardName, questionId }, callback) {
 
 describe("scenarios > dashboard > chained filter", () => {
   beforeEach(() => {
+    mockSessionProperty("field-filter-operators-enabled?", true);
```

Comment:
Do we need to do this if flag is actually enabled for Cypress?

Reply:
Oops, meant to remove that but forgot to press the save button apparently.